base/standard/src/main/java/org/openscience/cdk/stereo/StereoElementFactory.java

```diff
@@ -831,6 +834,10 @@ ExtendedCisTrans createExtendedCisTrans(List<IBond> dbs, Stereocenters stereocen
             if ((dbs.size() & 0x1) == 0)
                 return null;
 
+            for (IBond bond : dbs)
```

Comment:
Sure

Reply:
actually no, the input may be wrong so we can't trust the "bond.getStereo()" setting